src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.ConverterDataToObjImpl;
+import core.basesyntax.dao.ReaderDataFromFile;
+import core.basesyntax.dao.ReaderDataFromFileImpl;
+import core.basesyntax.db.DataBase;
+import core.basesyntax.service.ReportWriterToFileImpl;
+import core.basesyntax.service.TransactionStrategy;
+
+public class Main {
+    private static final String FILE_PATH = "src/main/resources/data.csv";
```

Comment:
its unclear for which file this constant serves
```suggestion
    private static final String INPUT_FILE_PATH = "src/main/resources/data.csv";
```